examples/index.html

```diff
@@ -28,12 +28,8 @@ <h2>friendly contenteditable API</h2>
         <h2 class="example-title">An editable paragraph</h2>
 
         <div class="paragraph-example example-sheet">
-          <p>
-            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer nec odio. Praesent libero. Sed ante dapibus diam. Sed nisi.
-          </p>
-          <p>
-            Nulla quis sem at nibh elementum imperdiet. Duis sagittis ipsum. Praesent mauris. Fusce nec tellus sed augue semper porta. Mauris massa. Vestibulum lacinia arcu eget nulla.
-          </p>
```

Comment:
With PR https://github.com/livingdocsIO/editable.js/pull/246 merged, this shouldn’t be a problem anymore?